backend/server/itconfig/log4j2.yaml

```diff
@@ -6,15 +6,19 @@ Configuration:
     Console:
       name: STDOUT
       target: SYSTEM_OUT
-      JsonTemplateLayout:
-        eventTemplateUri: "classpath:log-template.json"
-        locationInfoEnabled: true
-        EventTemplateAdditionalField:
-          key: "hostName"
-          value: "${env:hostName:-}"
-#      PatternLayout:
-#        pattern: "[%-5level] traceId=%X{trace_id} %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n"
+#      JsonTemplateLayout:
```

Comment:
Could this get a comment explaining why both layout types are here, and when you would need one vs. the other?

Reply:
Sure, I can do that. I do wonder if we should just get rid of the json option for running tests locally, since I doubt anyone actually likes it.

Reply:
done! thanks for the suggestion